### ISSUE TITLE
TOPページでのpropsのエラーについて

### ISSUE THREAD

**Issue by nobyn0312** [nobyn0312/jotai_todo#1]
疑問
Topページで、なぜここがエラーになるのかがわかりません。

```typescript
     <TodoList todos={todos} />
``` 

エラー文
```
型 '{ todos: any; }' を型 'IntrinsicAttributes' に割り当てることはできません。
  プロパティ 'todos' は型 'IntrinsicAttributes' に存在しません。ts(2322)
``` 

Todoのオブジェクトも宣言してるし、todos,setTodosにもuseState(Todos[])としてセットしています。

なるべく動画を見ないように作ってはいますが、難しかったので、
見て止めて、自分で書いてという感じでやってます。この動画だとだいたい23分ごろ。
https://youtu.be/nRCNL9T3J98?si=a0nneIYoMm8HEyI2

**Comment by nobyn0312** [comment]
https://github.com/nobyn0312/jotai_todo/assets/76238033/0a3063c7-63d0-45c4-a997-4c799be35ebc

受け取る側のTodoListでpropsを書いたらエラー消えました。
受け取り皿を作らないとダメって事なんでしょうか?

**Comment by yoshidev523** [comment]
そうですね、`Todo` コンポーネントにて、`TodoList` コンポーネントにtodosを渡しているのに、TodoListに用意されてないのはエラーになりますね。

コンポーネントは関数と同じです。引数が用意されてない関数に引数ありで呼び出すとエラーになりますよね、それと全く同じです！

```typescript
// 引数なしの関数
function sayHello() {
  console.log("Hello!");
}

// 引数はない関数を呼び出しているのでエラーになる
sayHello("Taro Yamada");
```
